Avoid crashes and wrong output on screenshot filter

diff --git a/ffmpeg/libswscale/swscale_internal.h b/ffmpeg/libswscale/swscale_internal.h
--- a/ffmpeg/libswscale/swscale_internal.h
+++ b/ffmpeg/libswscale/swscale_internal.h
@@ -1058,6 +1058,6 @@ void ff_init_vscale_pfn(SwsContext *c, yuv2planar1_fn yuv2plane1, yuv2planarX_fn
 #define MAX_LINES_AHEAD 4
 
 // enable use of refactored scaler code
-#define NEW_FILTER
+//#define NEW_FILTER
 
 #endif /* SWSCALE_SWSCALE_INTERNAL_H */
